src/transformers/models/splinter/configuration_splinter.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+# Copyright Ori Ram, Yuval Kirstain, Jonathan Berant, Amir Globerson, Omer Levy and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Splinter model configuration """
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+SPLINTER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "splinter-base": "https://huggingface.co/splinter-base/resolve/main/config.json",
+    "splinter-large": "https://huggingface.co/splinter-large/resolve/main/config.json",
```

Comment:
Those URLs do not work

Reply:
@sgugger can "tau/splinter-base" and "tau/splinter-large" work? 
I created an organization for Tel Aviv University at the model hub under "tau".

Reply:
Yes, you will just need to upload the model there :-)